package.json

```diff
@@ -40,6 +40,7 @@
     "gatsby-transformer-sharp": "^4.18.0",
     "gatsby-transformer-yaml": "^4.20.0",
     "latest": "^0.2.0",
+    "minimist": "^0.2.4",
```

Comment:
Where did this version come from? If I read it correctly, the package-lock.json file seems to show you need something higher than 1.2.0 for json5?

Reply:
That version came from the details of Dependbot alert in this repo.  In this PR's package-lock.json, you can see it will now use either 1.2.8 or 0.2.4, accordingly.